pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -580,6 +584,15 @@ func (r *ReconcileHyperConverged) ensureCDI(instance *hcov1alpha1.HyperConverged
 		}
 	}
 
+	if !reflect.DeepEqual(found.Spec, cdi.Spec) {
+		defaultUninstallStrategy := cdiv1alpha1.CDIUninstallStrategyBlockUninstallIfWorkloadsExist
```

Comment:
Maybe move this line into the `if` block?

Reply:
Sorry, I don't understand, do you mean inside `if found.Spec.UninstallStrategy == nil {` block?

Reply:
Right - sorry for the confusion.

Reply:
done